tezos_interop/tezos_interop.re

```diff
@@ -507,8 +507,6 @@ module Run_contract = {
   };
 };
 
-type michelson =
-  Tezos_micheline.Micheline.node(int, Michelson_v1_primitives.prim);
```

Comment:
Why did you remove this? Now the type is being duplicated in 3 places, please revert this change

Reply:
https://github.com/marigold-dev/sidechain/pull/158#discussion_r705558502